src/languageProvider/locales/ko-KR.json

```diff
@@ -42,7 +42,7 @@
   "create.cancel": "취소",
   "create.datePast": "지난 날짜는 선택할 수 없습니다.",
   "create.dateTime": "날짜 및 시간 선택하기",
-  "create.dialogTitle": "이벤트 만들기",
+  "create.dialogTitle": "Create an Event",
```

Comment:
don't we need Korean here?

Reply:
this should be the same translation. dont think there is uppercase in Korean.